nix/tests.nix

```diff
@@ -112,6 +113,58 @@ let
 
         ${withTmpDb postgresql} "$rootdir/test/memory-tests.sh"
       '';
+
+  coverage =
+    name: postgresql:
+    let
+      covCabalOptions="${devCabalOptions} --enable-coverage --enable-executable-dynamic";
+    in
+    checkedShellScript
+      name
+      ''
+        env="$(cat ${postgrest.env})"
+        export PATH="$env/bin:$PATH"
+
+        rootdir="$(${git}/bin/git rev-parse --show-toplevel)"
```

Comment:
@monacoremo Maybe we should refactor the rootdir / cd rootdir out. It seems like this is something that makes sense to do for every script.

Reply:
Yes, was also thinking about factoring out a postgrest-rootdir - lets do that in a separate PR. I added those for developer experience, as it's nice to have the scripts work consistently in nix shell independently from cwd